### PR TITLE
Added methods for /error/ID/done and /error/ID/false routes

### PR DESCRIPTION
**Pull request by PanierAvide**
Should do the job for #1

### CODE REVIEW

File: src/__tests__/index.test.js [hunk 1]
```diff
@@ -67,6 +69,35 @@ describe('Get Osmose errors', () => {
   });
 });
 
+describe('Update Osmose errors', async () => {
+  const language = 'fr';
+  const osmose = new OsmoseRequest({ language });
+  let errors = await osmose.fetchErrors({ item: 8120 });
```
Comment (GuillaumeAmat) [inline src/__tests__/index.test.js]: You should use `const` here

File: src/__tests__/index.test.js [hunk 2]
```diff
@@ -67,6 +69,35 @@ describe('Get Osmose errors', () => {
...
+
+  it('Should close error related to an Osmose item', async () => {
+    const errorId = errors[0].error_id;
+    const result = await osmose.closeError(errorId);
```
Comment (GuillaumeAmat) [inline src/__tests__/index.test.js]: You forgot to create a mock in the `src/__mocks__/requests.js` file. So you are actually calling the Osmose service and close an error :/

File: src/__tests__/index.test.js [hunk 3]
```diff
@@ -67,6 +69,35 @@ describe('Get Osmose errors', () => {
...
+    expect(closeErrorRequest).toBeCalledWith(
+      defaultOptions.endpoint,
+      language,
+      errorId
+    );
+  });
+
+  it('Should mark as false positive error related to an Osmose item', async () => {
+    const errorId = errors[1].error_id;
+    const result = await osmose.falseError(errorId);
```
Comment (GuillaumeAmat) [inline src/__tests__/index.test.js]: Same thing here about the mock :/

**Review by GuillaumeAmat — CHANGES_REQUESTED** [review]

**Comment by GuillaumeAmat** [comment]
Thank you @PanierAvide! o/

Your work is very clean, it misses mocks by the way, be careful to your tests until you create them.

**Comment by PanierAvide** [comment]
I will do the requested changes, I wasn't aware of the mocks (still learning modern JS !). I will update this soon, thanks for the review :-)

**Comment by PanierAvide** [comment]
Done ;-)

**Comment by GuillaumeAmat** [comment]
I think you misunderstood the purpose of mocks. They are here to replace real functions and so, acting like them.

In the current subject, the mocks have to prevent the call of real API routes. So your mock functions have to act like the real functions by returning proper results.

If you look at the previous mocks, they all return a JSON object that should be returned by the real API route. You have to do the same thing and not returning an html content telling us that the call raised an error.

**Comment by PanierAvide** [comment]
The problem is that for these routes, the Osmose API returns really this kind of response, and not a clean JSON. The fact that page says it's an error when showing `OK 200` is weird, particularly when you see that API has taken into account the change.

See `wget http://osmose.openstreetmap.fr/fr/api/0.2/error/<an error id>/done`

**Comment by GuillaumeAmat** [comment]
Oh crap... I think we should report it to the Osmose project, don't you think? That kind of behavior is not ideal in my opinion...

Anyway, you're PR is great, I merge it right now thank you!

**Review by GuillaumeAmat — APPROVED** [review]

**Comment by GuillaumeAmat** [comment]
Woops, I did not saw that the coverage decreased... I think there is a missing point.

**Comment by PanierAvide** [comment]
Well the API issue is probably related to https://github.com/osm-fr/osmose-backend/issues/259

File: src/__tests__/index.test.js [hunk 4]
```diff
@@ -67,6 +69,35 @@ describe('Get Osmose errors', () => {
...
+
+    expect(result).toBeDefined();
+    expect(result).toBeTrue();
+    defaultOptions.endpoint,
```
Comment (GuillaumeAmat) [inline src/__tests__/index.test.js]: There is a quirk here ^^

**Review by GuillaumeAmat — CHANGES_REQUESTED** [review]

File: src/__tests__/index.test.js [hunk 5]
```diff
@@ -67,6 +69,35 @@ describe('Get Osmose errors', () => {
   });
 });
 
+describe('Update Osmose errors', async () => {
```
Comment (GuillaumeAmat) [inline src/__tests__/index.test.js]: Ok got it! `describe` are not meant to be `async`.

File: src/__tests__/index.test.js [hunk 6]
```diff
@@ -67,6 +69,35 @@ describe('Get Osmose errors', () => {
...
+  const language = 'fr';
+  const osmose = new OsmoseRequest({ language });
+  const errors = await osmose.fetchErrors({ item: 8120 });
```
Comment (GuillaumeAmat) [inline src/__tests__/index.test.js]: ... So you have to fetch the errors in each `it` (not a perf problem since we have mocks)

**Review by GuillaumeAmat — CHANGES_REQUESTED** [review]

**Comment by GuillaumeAmat** [comment]
Furthermore, can you add your methods to the README?

File: src/__tests__/index.test.js (hunk 4 above)
Comment (GuillaumeAmat) [reply src/__tests__/index.test.js]: You still have the `defaultOptions.endpoint,` weirdness ;)

**Review by GuillaumeAmat — CHANGES_REQUESTED** [review]

**Review by GuillaumeAmat — APPROVED** [review]

**Comment by GuillaumeAmat** [comment]
The journey is over! ;)
Thanks again @PanierAvide

**Comment by PanierAvide** [comment]
Thanks for the review and integration :-)

**Comment by GuillaumeAmat** [comment]
Published in the 1.1.1 released :)

**Comment by frodrigo** [comment]
Osmose CORS issue should be fixed.

**Comment by PanierAvide** [comment]
@frodrigo Thank you :-)

**Comment by GuillaumeAmat** [comment]
Thanks @frodrigo

**Pull request merged**